<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetFindReplace</class>
 <widget class="QWidget" name="WidgetFindReplace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>87</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lineEditFind"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditReplace"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <widget class="QPushButton" name="pushButtonReplace">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Remplacer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonFind">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonReplaceAndFind">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Remplacer &amp; Rechercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonReplaceAll">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Remplacer Tout</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonClose">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>0</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxRegex">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Utiliser les expressions régulières</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxCasse">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Sensible à la casse</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
